module/styles/commons-importing-controller.less

```diff
@@ -60,6 +60,8 @@
   left: 180px;
 }
 
+.fbs-reset.fbs-pane.fbs-pane-type{width:561px}
```

Comment:
Nice! I would indent this as the rest of the file, spreading the CSS rule on multiple lines, with similar indentation (just for the sake of having a consistent style).